nixos/modules/services/web-servers/bluemap.nix

```diff
@@ -267,6 +266,12 @@ in {
             You can achieve this through setting `services.bluemap.eula = true`
           '';
         }
+        { assertion = cfg.enableNginx -> cfg.host != null;
+          message = ''
+            You have enabled nginx to serve the bluemap webapp, but have not assigned a host
+            to bluemap through setting `services.bluemap.host = "hostname"`
+          '';
+        }
```

Comment:
Alternatively if we just keep it undefined as I suggest in the option declaration we can just remove it entirely and keep things simple

```suggestion
```